WooCommerce/Classes/ViewRelated/Upgrades/UpgradesView.swift

```diff
@@ -438,6 +458,8 @@ private extension PurchaseUpgradeError {
 
 struct UpgradeWaitingView: View {
     let planName: String
+    var onViewLoaded: (() -> Void)
+    var onViewDismissed: (() -> Void)
```

Comment:
Why the naming difference here compared to `onDismiss` on the other views?

Reply:
No reason 😅